admin-dev/themes/new-theme/js/theme.js

```diff
@@ -56,12 +54,17 @@ import '@js/translation-page/index';
 
 import Header from '@js/header';
 
-new NavBar();
-new Header();
-
 import initDatePickers from '@js/app/utils/datepicker';
 import initInvalidFields from '@js/app/utils/fields';
 
+const {$} = global;
```

Comment:
The window object is the Global Object in the Browser
```suggestion
const {$} = window;
```

Reply:
I just apply lint in this pull request :sweat:

Reply:
Fixed by 46bc1e5aa256dc216e2158ad83830865e102ffd6

Reply:
<3